src/bda/plone/shop/user/userdata.py

```diff
@@ -166,11 +166,13 @@ class UserDataSchemaAdapter(AccountPanelSchemaAdapter):
 
     @property
     def fullname(self):
+        # TODO: I think this never gets called as there
+        # was an error in the string formatting.
```

Comment:
Have you tried adding a pdb? I'm not sure whether PAS just swallows the resulting AttributeError